packages/hydrogen-edge-adapter/src/server.js

```diff
@@ -0,0 +1,123 @@
+// Virtual entry point for the app
+import * as remixBuild from '@remix-run/dev/server-build'
```

Comment:
This should be converted to TypeScript like the other adapter files or is there a reason this one is a JS file?